test/probes/mysql.test.js

```diff
@@ -257,55 +171,45 @@ describe('probes.mysql ' + pkg.version, function () {
     })
   })
 
-  if (semver.gte(pkg.version, '2.0.0')) {
-    it('should create a pool and pool cluster', function () {
-      const poolConfig = {
-        connectionLimit: 10,
-        host: addr.host,
-        port: addr.port,
-        database: 'test',
-        user: user,
-        password: pass
-      }
+  it('should create a pool and pool cluster', function () {
+    const poolConfig = {
+      connectionLimit: 10,
+      host: addr.host,
+      port: addr.port,
+      database: 'test',
+      user: user,
+      password: pass
+    }
 
-      pool = db.pool = mysql.createPool(poolConfig)
-      cluster = db.cluster = mysql.createPoolCluster()
-      cluster.add(poolConfig)
-    })
-  }
+    pool = db.pool = mysql.createPool(poolConfig)
+    cluster = db.cluster = mysql.createPoolCluster()
+    cluster.add(poolConfig)
+  })
 
-  it('should sanitize SQL by default', function () {
+  it('should be configured to sanitize SQL by default', function () {
     ao.probes.mysql.should.have.property('sanitizeSql', true)
     // turn off for testing
     ao.probes.mysql.sanitizeSql = false
   })
 
   it('should trace a basic query', test_basic)
```

Comment:
since you are cleaning up ...
now that there wouldn't be repetition, would it make sense to change the pattern and instead of `function...` use the `it 'should ...'` pattern to have the goal of the test and the test code in one place

Reply:
I'm trying to move "horizontally" with test refactors, and slowly, because of the possibility of introducing a situation of a test pass without testing...

This as well as some linting and JS modernization, is on the todo list for after all features....